packages/tools/kadena-cli/src/account/commands/accountCreate.ts

```diff
@@ -0,0 +1,93 @@
+import { createSignWithKeypair } from '@kadena/client';
+import { createPrincipal } from '@kadena/client-utils/built-in';
+import { createAccount } from '@kadena/client-utils/coin';
+import type { ChainId } from '@kadena/types';
+import chalk from 'chalk';
+import type { Command } from 'commander';
+import debug from 'debug';
+import { FAUCET_CONSTANTS } from '../../constants/faucet.js';
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+
+export const createAccountCommand: (program: Command, version: string) => void =
+  createCommand(
+    'create',
+    'Create account',
+    [
+      globalOptions.fungible(),
+      globalOptions.network(),
+      globalOptions.chainId(),
+      globalOptions.publicKeys(),
+      globalOptions.predicate(),
+    ],
+    async (config) => {
+      debug('account-create:action')({ config });
+
+      try {
+        const principal = await createPrincipal(
+          {
+            keyset: {
+              pred: config.predicate as 'keys-all' | 'keys-2' | 'keys-any',
```

Comment:
The Type `'keys-all' | 'keys-2' | 'keys-any'` is used 4 times in this PR. That should probably be defined somewhere.

That said does `@kadena/client` not provide this type anywhere? I also noticed differences between for example `keys-2` and `keys-two` across branches. What does it really need to be?